spring-cloud-stream-module-launcher/README.md

```diff
@@ -16,9 +16,9 @@ cd ..
 2: copy the spring-cloud-stream source and sink sample JARs to `/opt/spring/modules`:
 
 ````
-mkdir -p /opt/spring/modules
-cp spring-cloud-stream/spring-cloud-stream-samples/source/target/spring-cloud-stream-sample-source-1.0.0.BUILD-SNAPSHOT-exec.jar /opt/spring/modules/time.jar
-cp spring-cloud-stream/spring-cloud-stream-samples/sink/target/spring-cloud-stream-sample-sink-1.0.0.BUILD-SNAPSHOT-exec.jar /opt/spring/modules/log.jar
+mkdir -p /opt/spring/modules/org/springframework/cloud/stream/module
+cp spring-cloud-stream/spring-cloud-stream-samples/source/target/spring-cloud-stream-sample-source-1.0.0.BUILD-SNAPSHOT-exec.jar /opt/spring/modules/org/springframework/cloud/stream/module
```

Comment:
Actually, regardless of what the local repo is, none of this is necessary since module-launcher will grab the JARs from the remote repo. We can remove all the unnecessary steps here instead (but I will submit a PR shortly that uses the standard local repo dir).